### DISCUSSION TITLE
Orange pi 5 "debian 11" install script multitoolbox.sh

### DISCUSSION THREAD

**Discussion by Anthony-landry** [RunOnFlux/flux#860] [Q&A]
Hello, 

is your multitoolbox.sh script compatible with debian 11 arm ?

if not can you add debian 11 

why : I have an orange pi 5 and only Debian 11 is available

Best Regards
G33k

**Comment by JKTUNING** [comment]
I also have an Orange Pi 5 that's on the way for testing. Debian should be supported in the installer though.

Mongo DB should install properly for Debian as it is supported and I just checked the code and it looks correct. I am hoping to test this within the week when it arrives!

**Comment by Anthony-landry** [comment]
hi, 

thank you for your answer 

I will try it 

best regards

**Comment by JKTUNING** [comment]
Here are some results for the Orange Pi 5 for CPU speed.

Cumulus results
```
sysbench cpu --threads=4 --cpu-max-prime=60000 --time=20 run
Running the test with following options:
Number of threads: 4

CPU speed:
    events per second:   851.43
```

Nimbus results
```
sysbench cpu --threads=8 --cpu-max-prime=60000 --time=20 run
Running the test with following options:
Number of threads: 8
CPU speed:
    events per second:  1174.07
```

**Comment by Anthony-landry** [comment]
hi , 

thanks you ;) 

sorry for my late response

best regards
